roles/create_vm_template/README.md

```diff
@@ -0,0 +1,69 @@
+Provision virtual machine
+================
+
+A role to provision a virtual machine, create associated resources if they don’t exist (subnets, vCPU, memory configuration, storage configuration, etc)
+This includes cloning and building from VM templates
+Boot a VM from these types:
+* Windows Server
+* RHEL8
+* RHEL9
+
+
+Requirements
+------------
+
+vCenter logged in permission
+
+Role Variables
+--------------
+
+* **vcenter_hostname** (str): (Required) 
```

Comment:
Let's use role prefix for variables. It's good practise. It's good practise to also accept collection name as prefix for variables. Not sure if it's valid for validated contant as well.